android/BOINC/app/src/main/java/edu/berkeley/boinc/attach/ProjectAttachService.java

```diff
@@ -524,7 +526,18 @@ else if(statusCredentials.error_num != BOINCErrors.ERR_OK) {
                 }
                 switch(statusCredentials.error_num) {
                     case BOINCErrors.ERR_DB_NOT_UNIQUE:
+                        Handler handler = new Handler(Looper.getMainLooper());
```

Comment:
'ERR_DB_NOT_UNIQUE' appears when trying to create an account only, so the message below will confuse user.
Please look at the errors and their description:
https://github.com/BOINC/boinc/blob/466e05dfd7e452373b80de89f096a1c477c0a8e8/android/BOINC/app/src/main/java/edu/berkeley/boinc/attach/ProjectAttachService.java#L409-L415